app/views/credits/credits_membership_success.ejs

```diff
@@ -0,0 +1,23 @@
+<!--
+Copyright © 2017, ACM@UIUC
+
+This file is part of the Groot Project.  
+
+The Groot Project is open source software, released under the University of
+Illinois/NCSA Open Source License.  You should have received a copy of
+this license in a file with the distribution.
+-->
+
+<%- include('../_partials/header') -%>
+
+<div id="meme-upload-container">
+  <div class="row col-centered small-12 medium-6 large-6">
+    <h3>ACM Membership Purchase Confirmation</h3>
+    <p>Thanks for submitting your purchase for an ACM membership! It may take a little bit to get your membership registered on our end, so we appreciate your patience. Once your membership processing is complete, you'll be able to access our intranet <a href="/intranet">here</a>.</p>
```

Comment:
change "you'll be able to access our intranet" to "you'll be able to access ACM's services" or something